client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -159,9 +149,13 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
             <tr>
               <td>Audit Type:</td>
               <td>
-                {auditType === 'BALLOT_POLLING'
-                  ? 'Ballot Polling'
-                  : 'Batch Comparison'}
+                {
+                  {
+                    BALLOT_POLLING: 'Ballot Polling',
+                    BATCH_COMPARISON: 'Batch Comparison',
+                    BALLOT_COMPARISON: 'Ballot Comparison',
+                  }[auditType]
+                }
```

Comment:
@jonahkagan Really cool way of doing this!!

Reply:
yeah i like this pattern cuz it makes every case explicit (as opposed to a series of ternary expressions, where the last case is implicit). still wishing that any of the languages we used had a multi-arm conditional (i.e. a `switch` that's an expression, not a statement)